src/docs/product/performance/apm-to-tracing.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: "Switching to @sentry/tracing"
+sidebar_order: 20
+---
+
+Our performance JavaScript SDK has been renamed from [`@sentry/apm`](https://www.npmjs.com/package/@sentry/apm) to [`@sentry/tracing`](https://www.npmjs.com/package/@sentry/tracing). The behaviour between `@sentry/apm` and `@sentry/tracing` is _identical_, but some imports and API have changed slightly.
+
+Although `@sentry/apm` is deprecated for `@sentry/tracing`, we will continue to support `@sentry/apm` with bug fixes for all `5.x.x` versions.
+
+## Migrating from @sentry/apm to @sentry/tracing
+
+For most cases, just replacing your `@sentry/tracing` import is enough to switch packages.
+
+```diff
+-import "@sentry/apm";
++import "@sentry/tracing";
+```
+
+If you were using the CDN use the tracing snippet instead:
+
+<JsCdnTag tracing />
+
+If you are using the automatic instrumentation for the _browser_, you should switch your imports from `Tracing` to `BrowserTracing`.
+
+```diff
+import * as Sentry from "@sentry/browser";
+-import { Integrations } from "@sentry/apm";
++import { Integrations } from "@sentry/tracing";
+
+ Sentry.init({
+   integrations: [
+-    new Integrations.Tracing(),
++    new Integrations.BrowserTracing(),
+   ]
+ });
+```
+
+If you were using the `beforeNavigate` option from the `Tracing` integration, the API for the `beforeNavigate` option in `BrowserTracing` has changed slightly. Instead of passing in a location and returning a string representing transaction name, `beforeNavigate` now accepts a transaction context and is expected to return transaction context. You can now add extra tags or change the op based on different parameters. If you want to access the location like before, you will have to manually look at `window.location`.
```

Comment:
This will need to change once we sunset `beforeNavigate` in favor of `tracesSampler`. Do we want to hold off publishing this until that happens?

Reply:
This is a good point. I will hold off until we make a decision in grooming tomorrow.

Reply:
Update after grooming: have decided to document `beforeNavigate`, but not to mention filtering.